ui-tests/tst/software/aws/toolkits/jetbrains/uitests/fixtures/WelcomeFrame.kt

```diff
@@ -29,4 +30,16 @@ class WelcomeFrame(remoteRobot: RemoteRobot, remoteComponent: RemoteComponent) :
             .findText(remoteRobot.preferencesTitle())
             .click()
     }
+
+    // Make a new project, not caring what it creates
+    // TODO change based off of IDE
```

Comment:
We could probably just make a temp folder and use Welcome Frame's OpenFolder action link.

That should work for any IDE

Reply:
Yep good idea, changed